src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs

```diff
@@ -438,6 +439,44 @@ public async Task<ActionResult> PartnershipDetails(PartnershipDetailsViewModel m
             return await CheckAuthorisedRepresentitiveAndRedirect();
         }
 
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> PartnerDetails(PartnerViewModel model, string action)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View(model);
+            }
+
+            if (model.PartnerModels.Count > maxPartnersAllowed)
+            {
```

Comment:
@seanciv I cleaner way of doing this is for PartnerViewModel to implement IValidatableObject, which will get called during model binding, but what you have done works...

Reply:
Thanks, even if its cleaner a few other hurdles came up that I dealt with before this. If its "fine" as is we can merge, if not, I can spend some time tomorrow on it